res/menu/profile_common.xml

```diff
@@ -2,7 +2,7 @@
 
 <menu xmlns:android="http://schemas.android.com/apk/res/android" xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <item android:title="@string/menu_edit_name"
+    <item android:title="@string/global_menu_edit_desktop"
```

Comment:
it is not always only the name.

depending on the chat, also the icon is editable.

and, in the near future, we will have the "Protect chat" setting here.

so, instead of saying in a complicated way _what_ can be edited, a simple "Edit" seems to be much easier to me.

Reply:
+1 for simple "Edit" that fits all kind of chat edition